projects/plugins/jetpack-boost/.github/workflows/integration.yml

```diff
@@ -0,0 +1,88 @@
+name: Integration
```

Comment:
This one might be worth moving into the monorepo's `.github/workflows/`. Unfortunately we don't have any infrastructure yet for running E2E tests.